### DISCUSSION TITLE
AIO page blank

### DISCUSSION THREAD

**Discussion by vanillyn** [nextcloud/all-in-one#2965] [Questions] [answered]
I recently wiped my server and I'm now attempting to load some of the data from nextcloud through a backup I created of the data, however after running nextcloud-aio-mastercontainer using the reverse-proxy command, the AIO page was blank:
![image](https://github.com/nextcloud/all-in-one/assets/108785622/f3ca42cd-86d4-4495-a34c-2a07f53c94b0)

I've never seen this and I couldn't find anything to help.

**Comment by szaimen** [comment, marked as answer]
Hi, I cannot reproduce this unfortunately. Do you maybe have something in the mastercontainer logs? Also what happens if you try to use a different browser?

**Reply by bitsn1per** [reply]
Hello! Same here. Previous installation was just standard AIO docker container, but at some point it has failed. So, now I'm trying to bring it up again with the backup and this time with NPM as proxy. After I've filled location for the backup with path and save it, it has opened new page, and then stucked. So, now every time I'm running it I see the same blank page, even though  all the logs for the mastercontainer and domaincheck container looks good. Please, check below:

Mastercontainer:
"Trying to fix docker.sock permissions internally...
Creating docker group internally with id 995
The environmental variable APACHE_DISABLE_REWRITE_IP has been set which is not supported by AIO. Please remove it!
The environmental variable NEXTCLOUD_TRUSTED_DOMAINS has been set which is not supported by AIO. Please remove it!
The environmental variable TRUSTED_PROXIES has been set which is not supported by AIO. Please remove it!
Initial startup of Nextcloud All-in-One complete!
You should be able to open the Nextcloud AIO Interface now on port 8080 of this server!
E.g. https://internal.ip.of.this.server:8080
If your server has port 80 and 8443 open and you point a domain to your server, you can get a valid certificate automatically by opening the Nextcloud AIO Interface via:
https://your-domain-that-points-to-this-server.tld:8443
[Mon Nov 13 13:42:54.546647 2023] [mpm_event:notice] [pid 122:tid 140612395526984] AH00489: Apache/2.4.58 (Unix) OpenSSL/3.1.4 configured -- resuming normal operations
[Mon Nov 13 13:42:54.547080 2023] [core:notice] [pid 122:tid 140612395526984] AH00094: Command line: 'httpd -D FOREGROUND'
[13-Nov-2023 13:42:54] NOTICE: fpm is running, pid 128
[13-Nov-2023 13:42:54] NOTICE: ready to handle connections
INF ts=1699882974.5611358 msg=using provided configuration config_file=/Caddyfile config_adapter="

Domaincheck:
"2023-11-13 13:44:30: (../src/server.c.1933) server started (lighttpd/1.4.73)"

**Reply by szaimen** [reply]
Interesting! Can you post the config file here? Please make sure to mask any sensitive data. You can get the config with `sudo docker run -it --rm --volume nextcloud_aio_mastercontainer:/mnt/docker-aio-config:ro alpine sh -c "cat /mnt/docker-aio-config/data/configuration.json"`

**Reply by bitsn1per** [reply]
Wow! You just like a thunder. Please, find that output:

{
    "password": "here_is_my_great_password",
    "nextcloud_datadir": "/media/b1t/T51/ncdata",
    "secrets": {
        "INSTANCE_ID": "here_is_my_instance_id",
        "NEXTCLOUD_PASSWORD": "here_is_my_great_password",
        "BORGBACKUP_PASSWORD": "here_is_my_another_pass"
    },
    "apache_port": "11000",
    "apache_ip_binding": "0.0.0.0",
    "borg_backup_host_location": "/media/b1t/backup",
    "borg_restore_password": "here_is_my_super-duper",
    "instance_restore_attempt": 0,
    "backup-mode": "test",
    "nextcloud_mount": "/media/"

**Reply by szaimen** [reply]
Interesting! is there no closing bracket at the end of the file or did you forget to copy it?

**Reply by bitsn1per** [reply]
I have noticed too, and even have re-selected from the first bracket to the last symbol very carefully.

**Reply by szaimen** [reply]
So there is actually no closing bracket?

**Reply by szaimen** [reply]
Hi, can you please upgrade to 7.7.0 Beta and check if there are some logs shown in the mastercontainer after the upgrade? See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel
